parkings/tests/api/conftest.py

```diff
@@ -57,3 +57,13 @@ def operator_2_api_client(operator_2):
     token_authenticate(api_client, operator_2.user)
     api_client.operator = operator_2
     return api_client
+
+
+@pytest.fixture
+def operator_enforcer(enforcer_factory, operator):
+    return enforcer_factory(user=operator.user)
+
+
+@pytest.fixture
+def operator_enforcer2(enforcer_factory, operator_2):
+    return enforcer_factory(user=operator_2.user)
```

Comment:
These two shouldn't be needed as there should be no operators that are also enforcers.